try:
    print(a)
except:
    print("This is an exception!")
